biogrid/apps/biogrid-mvp/src/app/pages/input-page/input-page.css

```diff
@@ -0,0 +1,53 @@
+.input-page {
+  background-color: #f2f2f2;
+  border-radius: 0.3125rem;
+  font-family: sans-serif;
+  margin: auto;
+  padding: 1.25rem;
+  text-align: center;
+  width: 56.25rem;
+}
+
+body {
+  background-color: #3b6d6d;
+}
+
+br {
+  clear: both;
+}
+
+img {
```

Comment:
nit: this styling looks like its meant to apply to a specific image, might be best to give the img div a class and reference it in the css through the class name as opposed to applying this styling to all images
(this works here because you only have one image, but good to set up now in case more images are added)